sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_cbs_async.py

```diff
@@ -154,7 +169,7 @@ async def _on_execute_operation_complete(
     async def _update_status(self):
         if self.auth_state in (CbsAuthState.OK, CbsAuthState.REFRESH_REQUIRED):
```

Comment:
could you do `and self._expires_on` here to avoid the cast below? or we'd want to pass it in even if it's None here?